rasa/shared/data.py

```diff
@@ -175,16 +158,7 @@ def is_story_file(file_path: Text) -> bool:
     Returns:
         `True` if it's a story file, otherwise `False`.
     """
-    from rasa.shared.core.training_data.story_reader.yaml_story_reader import (
-        YAMLStoryReader,
-    )
-    from rasa.shared.core.training_data.story_reader.markdown_story_reader import (
-        MarkdownStoryReader,
-    )
-
-    return YAMLStoryReader.is_stories_file(
-        file_path
-    ) or MarkdownStoryReader.is_stories_file(file_path)
+    return YAMLStoryReader.is_stories_file(file_path)
 
 
 def is_test_stories_file(file_path: Text) -> bool:
```

Comment:
do we still need these functions which just call another function or can we also drop them? I think I'd drop them as we are not expecting a new format soon